boards/arm/ev11l78a/doc/index.rst

```diff
@@ -0,0 +1,73 @@
+.. _ev11l78a:
+
+UPD301C Basic Sink Application Example
+######################################
+
+Overview
+********
+
+The UPD301C Basic Sink Application Example Evaluation Kit is a
+low-cost evaluation platform for Microchip's UPD301C Standalone
+Programmable USB Power Delivery (PD) Controller. This RoHS-compliant
+evaluation platform comes in a small form factor and adheres to the
+USB Type-C™ Connector Specification and USB PD 3.0 specification.
+
```

Comment:
Add an image of the board? (⚠️ It needs to be <50KB)

Reply:
I've got a problem for this one — I won't be able to get a good picture of the board myself at least on short order.

I see that the SAM D20 just took it off Microchip's website, but it's not my copyright so I wouldn't be able to send the pull request then 😅

Might need someone from Microchip to contribute that one if they can?

Reply:
There's no issue using one of the "official" pictures, as it's considered fair use. We do it for pretty much every board :) Just make sure to credit the photo (again, you can see how it's done for other boards). Thanks!